hydra-plutus/src/Hydra/ContractSM.hs

```diff
@@ -123,121 +74,3 @@ typedValidator threadToken =
    in Scripts.mkTypedValidator @(StateMachine State Input)
         val
         $$(PlutusTx.compile [||wrap||])
-
--- | The machine client of the hydra state machine. It contains both, the script
--- instance with the on-chain code, and the Haskell definition of the state
--- machine for off-chain use.
-machineClient ::
-  -- | Thread token of the instance
-  AssetClass ->
-  StateMachineClient State Input
-machineClient threadToken =
```

Comment:
We moved parts of this code together on `master` .. didn't we? However I do not think we should move all of it (seeing it now here).